### ISSUE TITLE
Confusing message when bad COM port given

### ISSUE THREAD

**Issue by PropGit** [dbetz/proploader#20]
When an invalid COM port is given:
```
$ proploader -p COM2 Welcome.elf
error: loader initialization failed: -1
```
The error that appears looks like some kind of catch-all error for the loader, but when I expected is something to indicate it couldn't open or communicate on that port.

As in Issue #19, please make this case generate the new error message ```110-Unable to connect to port: %s\n``` instead, followed by a 102 error message (if possible).

**Comment by PropGit** [comment]
Tested v1.0_8.  @dbetz - The "Unable to connect..." message is a -v message.  Please make it a normal (non -v) message.  Also, it appears with a code of 000... this may be only because of the -v, I don't know, but please make it 110 as it's the same message as in Issue #19.

**Comment by PropGit** [comment]
Fixed in v1.0_11.

Verified correct; message is a 117 or 118 depending on the type of port (COM or Wi-Fi, respectively).
